src/utils/organization.tsx

```diff
@@ -21,3 +21,11 @@ export const isAdmin = (organization: ServerOrganization | undefined) => {
 export const isContributor = (roleHolder: ServerOrganization | OrganizationUser | undefined) => {
   return roleHolder?.role === 'Contributor';
 };
+
+export const getAllNurseries = (organization: ServerOrganization): (Facility | undefined)[] => {
```

Comment:
small request, since getAllSeedBanks and getAllNurseries are similar code.. can we extract out the common code into a function?
```
getFacilitiesByType = (organization, type) => // returns facilities by type
getAllSeedBanks = (organization) => getFacilitiesByType(organziation, 'Seed Bank');
getAllNurseries = (organization) => getFacilitiesByType(organization, 'Nursery');
```